cost/superseded_instance_remediation/superseded_instance_remediation.pt

```diff
@@ -0,0 +1,304 @@
+      field "instance_type", jmes_path(col_item, "links.instance_type.name")
+      field "region", jmes_path(col_item, "links.cloud.region")
+      field "cloud_name", val(iter_item, "display_name")
+      field "locked", jmes_path(col_item, "locked")
+      field "state", jmes_path(col_item, "state")
+    end
+  end
+end
+
+datasource "ds_combined_instance_data" do
+  run_script $js_merge_instance_data, $ds_instances_16, $param_old_instance_type
+end
+
+###############################################################################
+# Scripts
+###############################################################################
+
+script "js_instances_16", type: "javascript" do
+  result "request"
+  parameters "rs_cm_host","cloud_href","rs_project_id","param_exclude_tags"
+  code <<-EOS
+    var filtered_tags = ""
+    console.log(param_exclude_tags[0])
+    if ( param_exclude_tags.length > 0 && param_exclude_tags[0] != "" ) {
+      var filtered_tags = '&tag!=' + param_exclude_tags.join(",");
+    }
+
+    request = {
+      "auth": "auth_rs",
+      "verb": "GET",
+      "host": rs_cm_host,
+      "path": cloud_href + "/instances",
+      "headers": {
+        "X-Api-Version": "1.6",
+        "X-Account": \""+rs_project_id+\""
+      },
+      "query_params": {
+        "view": "full",
+        "filter": 'state=provisioned,operational,running' + filtered_tags
+      }
+    }
+
+  EOS
+end
+
+script "js_merge_instance_data", type: "javascript" do
+  parameters "ds_instances", "param_old_instance_type"
+  result "results"
+  code <<-EOS
+    var results = []
+    for ( i=0; i < ds_instances.length; i++) {
+      var instance = ds_instances[i]
+      if ( instance.instance_type == param_old_instance_type ){
+        results.push(instance)
+      }
+    }
+  EOS
+end
+
+###############################################################################
+# Escalations
+###############################################################################
+escalation "esc_resize_instances" do
+  request_approval  do
+    label "Escalation approval"
+    description "Approve resize action"
+    parameter "approval_reason" do
+      type "string"
+      label "Reason for approval"
+      description "Explain why you are approving the action"
+    end
+  end
+
+  run "resize_instances", data, $param_new_instance_type
+  email $param_email
+end
+
+resolution "res_resize_instances" do
+  email $param_email
+end
+
+###############################################################################
+# Policy
+###############################################################################
+
+policy "policy_superseded_instance" do
+  validate $ds_combined_instance_data do
+    summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} Instances will be resized from {{ parameters.param_old_instance_type }} to {{ parameters.param_new_instance_type }}"
+    detail_template <<-EOS
+# Superseded Instances
+| Resource UID | Name | Cloud |
```

Comment:
include current and new instance type?  should the row indicate that an instance type can not be changed.